acceptance/bundle/resource_deps/jobs_update/script

```diff
@@ -0,0 +1,45 @@
+print_requests() {
+    jq --sort-keys 'select(.method != "GET" and (.path | contains("/jobs")))' < out.requests.txt
+    rm out.requests.txt
+}
+
+print_sorted_requests() {
+    jq -c < out.requests.txt | sort | jq --sort-keys 'select(.method != "GET" and (.path | contains("/jobs")))'
+    rm out.requests.txt
+}
+
+echo "*" > .gitignore
+# In direct, plan is sorted by deployment order. In TF it comes back sorted alphabetically.
+# I think deployment order is better, sorting here to remove the mismatch
```

Comment:
This sorting can be removed once TF support is gone entirely.

Should we track a list of changes we can/should make for when that is the case?